testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -331,22 +362,143 @@ pub fn clean_k8s_cluster(
     // parse genesis
     let helm_values = String::from_utf8(raw_helm_values.stdout).unwrap();
     let v: Value = serde_json::from_str(&helm_values).unwrap();
-    let era = &v["genesis"]["era"];
-    let num_validators = &v["genesis"]["numValidators"];
-    let new_era = if era == 1 { 2 } else { 1 };
+    let chain_era = v["genesis"]["era"].as_i64().expect("not a i64") as usize;
+    let curr_num_validators = v["genesis"]["numValidators"].as_i64().expect("not a i64") as usize;
 
-    println!("genesis.era: {} --> {}", era, new_era);
+    // get the new era
+    let mut rng = rand::thread_rng();
+    let new_era = rng.gen::<u32>() as usize;
+    println!("genesis.era: {} --> {}", chain_era, new_era);
     println!(
         "genesis.numValidators: {} --> {}",
-        num_validators, base_num_validators
+        curr_num_validators, base_num_validators
     );
 
+    // scale down. helm uninstall validators while keeping history for later
+    (0..MAX_NUM_VALIDATORS).into_par_iter().for_each(|i| {
```

Comment:
why do we not use base_num_validators here? we can not guaranty cluster always has 30 validators?

Reply:
We're removing all 30 helm releases since helm keeps the history. There's no guarantee that the cluster will have 30 validators on each test run though, since the idea is to give tests themselves the ability to scale up/down the cluster.